src/classes/HH_HouseholdNaming_TEST.cls

```diff
@@ -109,50 +109,106 @@ public class HH_HouseholdNaming_TEST {
                     npo02__Excluded_Account_Opp_Rectypes__c = null,
                     npo02__Excluded_Account_Opp_Types__c = null,
                     npo02__Excluded_Contact_Opp_Rectypes__c = null,
-                    npo02__Excluded_Contact_Opp_Types__c = null                    
+                    npo02__Excluded_Contact_Opp_Types__c = null
                 ));
-                
+
             npe01__Contacts_and_Orgs_Settings__c contactSettingsForTests = UTIL_CustomSettingsFacade.getContactsSettingsForTests(new npe01__Contacts_and_Orgs_Settings__c (
                 npe01__Account_Processor__c = CAO_Constants.ONE_TO_ONE_PROCESSOR,
                 npe01__Enable_Opportunity_Contact_Role_Trigger__c = true,
                 npe01__Opportunity_Contact_Role_Default_role__c = 'Donor'
             ));
-        
+
         integer i;
-        
+
         list<Contact> insertlist = new list<Contact>();
         for (i = 0; i < 20; i++){
             Contact c = new Contact(LastName = 'LastName' + i, FirstName = 'FirstName' + i, Salutation = 'Mr.');
```

Comment:
This variable doesn't seem to have a very insightful name. Please give variables names that suggest their purpose.